atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -31,6 +35,9 @@
             FunctionCheckedException<Cassandra.Client, V, K> fn) throws K;
     InetSocketAddress getAddressForHost(String host) throws UnknownHostException;
     <V, K extends Exception> V runWithRetry(FunctionCheckedException<Cassandra.Client, V, K> fn) throws K;
+    Map<InetSocketAddress, CassandraClientPoolingContainer> getCurrentPools();
     InetSocketAddress getRandomHostForKey(byte[] key);
+    RangeMap<CassandraClientPoolImpl.LightweightOppToken, List<InetSocketAddress>> getTokenMap();
```

Comment:
Hmm, the interface depends on the impl's inner-class (though it's static) which seems awkward. Maybe `LightweightOppToken` could be its own class, if we need to preserve this structure? Also the notion of blacklisting hosts feels like an implementation detail (to me at least)

Reply:
Agree, this is awkward. Luckily, this is only needed for tests so we were able to get rid of it form the interface.